src/lib/blueprint.ts

```diff
@@ -307,7 +311,12 @@ const createRoutes = async (
     }
   }
 
-  return Array.from(routeMap.values())
+  let routes = Array.from(routeMap.values())
+
+  routes = updateRoutesStatus(routes)
```

Comment:
Let's avoid generic function names and unnecessary let bindings. "Update" functions suggest this mutates in place anyway. It looks like we could do this with a simple map function, e.g.,

```
routes.map(addIsDeprecatedToRoute).map(addIsUndocumentedToRoute)
```

Reply:
aff41ae738b1be51580075cb3b0820484bdf1b3a